internal/gcsx/bucket_manager.go

```diff
@@ -222,6 +222,9 @@ func (bm *bucketManager) SetUpBucket(
 		bm.config.TmpObjectPrefix,
 		b)
 
+	// Fetch and set bucket type
+	b.FetchAndSetBucketType()
```

Comment:
This will require additional permission to the user while mounting, right?

Do we need to update any documentation or playbook?